<launch>
    <node pkg="orb_slam3" type="mo_stereo" name="mo_stereo"
        args="$(find orb_slam3)/Vocabulary/ORBvoc.bin $(find orb_slam3)/METOAK/U60.yaml /dev/video0"
        output="screen">
        <param name="trajectoryFileLocation" type="string" value="/home/young/vslam_ros/trajectory/" />
    </node>
</launch>